src/App.js

```diff
@@ -1,25 +1,60 @@
-import logo from './logo.svg';
 import './App.css';
+import { Component } from 'react';
+import SearchForm from './Components/SearchForm';
+import ResultsDisplay from './Components/ResultsDisplay';
+import axios from 'axios';
+import Card from 'react-bootstrap/Card';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      location: {},
+      error: '',
+    };
+  }
+
+  getLocation = async (q) => {
+    let API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_KEY}&q=${q}&format=json`;
+    let errorFound = false;
```

Comment:
Nice work so far! 

For `getLocation()` I would reconsider the param name of `q` so this is a bit more readable. We know that it is a param so this is something to consider for next time.

Reply:
Will do, thank you Riva!